TheFourFifths/consus
# New Server Functionality for getting faulty items

### Summary

Server endpoint for getting all of the faulty items.

### Checklist


### How to Review

1. Use the client to set up some items with faults. 
2. Run the server
3. Ensure "/item/fault/all" returns the faulty items

### Links

- [TFF-200](https://msoese.atlassian.net/browse/TFF-200)
